### PR TITLE
Use MCTruthContainer in the CA tracker

### PR DESCRIPTION
**Pull request by mpuccio**
Dear @shahor02,
with @mconcas we rebased #6431 onto the dev branch. Just to be clear (for GitHub at least), this closes #6431.
Cheers,
Matteo and Max

### CODE REVIEW

**Review by mconcas — DISMISSED** [review]

**Review by mconcas — DISMISSED** [review]

File: Detectors/ITSMFT/ITS/tracking/src/IOUtils.cxx [hunk 1]
```diff
@@ -78,55 +78,55 @@ void ioutils::convertCompactClusters(gsl::span<const itsmft::CompClusterExt> clu
   }
 }
 
-std::vector<ROframe> ioutils::loadEventData(const std::string& fileName)
```
Comment (shahor02) [inline Detectors/ITSMFT/ITS/tracking/src/IOUtils.cxx:81]: can this be removed?

**Review by shahor02 — DISMISSED** [review]
Thanks, just a small comment below.

File: Detectors/ITSMFT/ITS/tracking/src/IOUtils.cxx (hunk 1 above)
Comment (mpuccio) [reply Detectors/ITSMFT/ITS/tracking/src/IOUtils.cxx:81]: Good point, removed.

**Comment by mpuccio** [comment]
uhm... osx is taking longer than expected (36h) and now also dataflow is spinning again... why? @TimoWilken

**Comment by TimoWilken** [comment]
> uhm... osx is taking longer than expected (36h)

The OSX O2 builder was slowed down recently, I've changed the way it's run and the number of parallel compilation jobs was increased today, so the builder should catch up soon.

> and now also dataflow is spinning again... why? @TimoWilken

Earlier today, there was an issue with a few builders stepping on each other. Now fixed.

**Comment by mpuccio** [comment]
osx failures look to me unrelated

**Pull request merged**